runtime/src/lib.rs

```diff
@@ -362,6 +357,10 @@ impl pallet_balances::Config for Runtime {
 	type ExistentialDeposit = ConstU128<EXISTENTIAL_DEPOSIT>;
 	type AccountStore = System;
 	type WeightInfo = pallet_balances::weights::SubstrateWeight<Runtime>;
+	type HoldIdentifier = ();
+	type FreezeIdentifier = ();
+	type MaxHolds = ();
+	type MaxFreezes = ();
 }
```

Comment:
We have found out meanwhile that we need to set them to `Const<0>`. :)

Reply:
All of them? Including the ones previously defined ? (MaxReserves, DustRemoval, ... )

Reply:
nope just `MaxHolds` and `MaxFreezes`